index.html

```diff
@@ -9,42 +9,39 @@
   </head>
   <body>
     <h1 class="title">Calculator</h1>
-    <div class="calc-background">
-      <section class="calc-row">
-        <p class="answer-number">0</p>
-      </section>
+    <article class="calculator">
```

Comment:
Please address the issue of excess space at the bottom of the calculator

Reply:
Border of the calculator is thinner that spaces between the buttons. Please make outer border at least as wide than inner ones.

Reply:
The buttons are not exactly square

<img width="267" alt="Screenshot 2021-06-09 at 23 03 16" src="https://user-images.githubusercontent.com/28704906/121429242-fc3b4a00-c976-11eb-8243-4ad099adadd4.png">